test_util/launch_configs/README.md

```diff
@@ -0,0 +1,46 @@
+# DC/OS Launch Configuration YAML
+## Design Intention
```

Comment:
There will be a required versioning field to explicitly link to the configuration schema required in the given version of launch